app/packages/app/src/index.tsx

```diff
@@ -137,7 +169,7 @@ const App: React.FC = ({}) => {
     );
 
     return () => controller.abort();
-  }, []);
+  }, [clearModal, handleError, refresh, screenshot, subscription]);
```

Comment:
@kaixi-wang I am removing these dependencies - they seem to cause the samples modal **not** to open.